docs_src/microservices/security/Dockerfile-remote-sshd

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y openssh-server
+RUN mkdir /var/run/sshd
+RUN echo 'root:THEPASSWORDYOUCREATED' | chpasswd
+RUN sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
```

Comment:
More detail needed.

Reply:
see details on this  https://docs.docker.com/engine/examples/running_ssh_service/  this is the same one I used.

Reply:
Please add comment that refers to this: https://stackoverflow.com/questions/21391142/why-is-it-needed-to-set-pam-loginuid-to-its-optional-value-with-docker

Reply:
ok